tests/integration/api_v3/http/posStakes.test.js

```diff
@@ -32,46 +32,35 @@ const endpoint = `${baseUrlV3}/pos/stakes`;
 describe('Stakes API', () => {
 	let refValidator;
 	beforeAll(async () => {
-		let response;
 		do {
 			// eslint-disable-next-line no-await-in-loop
-			response = await api.get(`${baseUrlV3}/dpos/delegates?limit=1`);
-		} while (!response.data);
-		[refValidator] = response.data;
+			const { data: [stakeTx] = [] } = await api.get(`${baseUrlV3}/transactions?moduleCommand=pos:stake&limit=1`);
+			if (stakeTx) {
+				refValidator = stakeTx.sender;
+			}
+		} while (!refValidator);
 	});
 
-	// TODO: Add missing tests similar to stakers
 	describe(`GET ${endpoint}`, () => {
 		it('Returns list of sent stakes when requested for known staker address', async () => {
 			const response = await api.get(`${endpoint}?address=${refValidator.address}`);
 			expect(response).toMap(stakeResponseSchema);
-			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
-			expect(response.data.votes.length).toBeLessThanOrEqual(10);
+			expect(response.data.stakes.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.stakes.length).toBeLessThanOrEqual(10);
```

Comment:
Can we please fetch the `maxNumberSentStakes` information from the `/api/v3/pos/constants` endpoint and cache it in the `beforeAll` hook and use it as the highest count in all the test cases?
```suggestion
			expect(response.data.stakes.length).toBeLessThanOrEqual(maxNumberSentStakes);
```